src/warehouse/index.js

```diff
@@ -66,6 +67,7 @@ const rudderReservedColums = {
   screen: { ...whDefaultColumnMappingRules, ...whScreenColumnMappingRules },
   group: { ...whDefaultColumnMappingRules, ...whGroupColumnMappingRules },
   alias: { ...whDefaultColumnMappingRules, ...whAliasColumnMappingRules },
+  extract: { ...whExtractEventTableColumnMappingRules }
```

Comment:
Just wanted to confirm, we don't need these right?
```Go
const rules = {
  id: 'messageId',
  anonymous_id: 'anonymousId',
  user_id: 'userId',
  sent_at: 'sentAt',
  timestamp: 'timestamp',
  received_at: 'receivedAt',
  original_timestamp: 'originalTimestamp',
  channel: 'channel',
  context_ip: (message) => getFirstValidValue(message, ['context.ip', 'request_ip']),
  context_request_ip: 'request_ip',
  context_passed_ip: 'context.ip',
};
```

Reply:
Yeah, we kept `id` and `receivedAt` for extract events